### ISSUE TITLE
async aws clients

### ISSUE THREAD

**Issue by JakubKahovec** [seratch/AWScala#62]
Hi,
do you plan to support async aws clients (i.e AmazonDynamoDBAsyncClient etc) ?
Thanks
Jakub

**Comment by seratch** [comment]
>do you plan to support async aws clients (i.e AmazonDynamoDBAsyncClient etc) ?

No, I don't. If other collaborators will work on this, that would be great.
